rest_framework/routers.py

```diff
@@ -254,7 +254,7 @@ def get_urls(self):
                 #   so a slash in the beginning will (A) cause Django to give
                 #   warnings and (B) generate URLS that will require using '//'.
                 if not prefix and regex[:2] == '^/':
-                    regex = '^' + regex[2:]
+                    regex = f'^{regex[2:]}'
```

Comment:
This particular case reads more obviously to me as a plain ol' string concatenation.

(I suppose because it's building a regex, and I have to stop and think hard about if the curly-braces and square brackets here are a reg-ex or part of an f-string.)

Reply:
changed back